rebasehelper/options.py

```diff
@@ -0,0 +1,199 @@
+    },
+    {
+        "name": ("--not-download-sources",),
+        "default": False,
+        "switch": True,
+        "help": "do not download sources"
+    },
+    {
+        "name": ("-w", "--keep-workspace"),
+        "default": False,
+        "switch": True,
+        "help": "do not remove workspace directory after finishing"
+    },
+    {
+        "name": ("-c", "--continue"),
+        "default": False,
+        "switch": True,
+        "dest": "cont",
+        "help": "continue previously interrupted rebase"
+    },
+    {
+        "name": ("--color",),
+        "choices": ["always", "never", "auto"],
+        "default": "auto",
+        "help": "colorize the output, defaults to %(default)s"
+    },
+    {
+        "name": ("--disable-inapplicable-patches",),
+        "default": False,
+        "switch": True,
+        "dest": "disable_inapplicable_patches",
+         "help": "disable inapplicable patches in rebased SPEC file"
+    },
+    {
+        "name": ("--non-interactive",),
+        "default": False,
+        "switch": True,
+        "dest": "non_interactive",
+        "help": "do not interact with user"
+    },
+    {
+        "name": ("--build-tasks",),
+        "dest": "build_tasks",
+        "metavar": "OLD_TASK,NEW_TASK",
+        "type": lambda s: s.split(','),
+        "help": "comma-separated remote build task ids"
+    },
+    {
+        "name": ("--builds-nowait",),
+        "default": False,
+        "switch": True,
+        "help": "do not wait for remote builds to finish"
+    },
+    {
+        "name": ("--builder-options",),
+        "default": None,
+        "metavar": "BUILDER_OPTIONS",
+        "help": "enable arbitrary local builder option(s), enclose %(metavar)s in quotes "
+                "and note that = before it is mandatory"
+    },
+    {
+        "name": ("--srpm-builder-options",),
+        "default": None,
+        "metavar": "SRPM_BUILDER_OPTIONS",
+        "help": "enable arbitrary local srpm builder option(s), enclose %(metavar)s in quotes "
+                "and note that = before it is mandatory"
+    },
+    {
+        "name": ("--build-retries",),
+        "default": 2,
+        "help": "number of retries of a failed build, defaults to %(default)d",
+        "type": int
+    },
+    {
+        "name": ("--results-dir",),
+        "help": "directory where rebase-helper output will be stored"
+    },
+    {
+        "name": ("--get-old-build-from-koji",),
+        "default": False,
+        "switch": True,
+        "help": "do not build old sources, download latest build from Koji instead"
+    },
+    {
+        "name": ("sources",),
+        "metavar": "SOURCES",
+        "nargs": "?",
+        "default": None,
+        "help": "new upstream sources"
+    },
+    {
+        "name": ("--changelog-entry",),
+        "default": "- New upstream release %{version}",
+        "help": "text to use as changelog entry, can contain RPM macros, which will be expanded"
+    }
```

Comment:
There should always be a comma after last item of a list. It has no disadvantages and enables you to add more items without caring whether there is a comma or not. Also there are no unnecessary comma adding lines in diff.
The same applies of course for dict items.